src/components/Clock/Clock.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export class Clock extends React.Component {
+  state = {
+    time: new Date(),
+  };
+
+  timer = setInterval(() => {
+    this.setState({
+      time: new Date(),
+    });
+
+    // eslint-disable-next-line
+    console.log(new Date().toLocaleTimeString());
+  }, 1000);
```

Comment:
setInterval in ComponentDidMount
https://reactjs.org/docs/state-and-lifecycle.html#adding-lifecycle-methods-to-a-class

Reply:
Let's follow Alisa's advice